interfaces/cython/cantera/composite.py

```diff
@@ -628,7 +634,7 @@ def sort(self, col, reverse=False):
             indices = indices[::-1]
         self._states = [self._states[ix] for ix in indices]
         for k, v in self._extra.items():
-            self._extra[k] = list(np.array(v)[indices])
+            self._extra[k] = np.array(v)[indices]
```

Comment:
These should already all be arrays, right? So no need to do the `np.array(v)` conversion?

Reply:
yes, this is redundant. was relevant to the original issue somehow I guess